src/main/scala/com/scalapenos/riak/RiakBucket.scala

```diff
@@ -70,5 +70,9 @@ trait RiakBucket {
   def lastWriteWins_=(value: Boolean): Future[Unit] = setLastWriteWins(value)
   def setLastWriteWins(value: Boolean): Future[Unit] = setProperties(Set(LastWriteWins(value)))
 
-}
+  // ============================================================================
+  // Special operations (Use with care)
+  // ============================================================================
 
+  def allKeys(): Future[RiakKeys]
```

Comment:
What if a `RiakBucket` had an `unsafe` method that produced an instance of `UnsafeBucketOperations` that you could then call `allKeys()` on? Would that be too protective?